integreat_cms/cms/templates/poicategories/poicategory_form.html

```diff
@@ -19,6 +19,43 @@ <h1 class="heading">
         </div>
         <div class="flex flex-wrap">
             <div class="w-full">
+                <div class="xl:flex gap-3 pt-2">
+                    <div class="xl:w-1/2 mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
+                        <div class="w-full p-4 rounded bg-water-500">
+                            <h3 class="heading font-bold text-black">
+                                <i icon-name="tag" class="mr-2"></i> {% translate "Category icon" %}
+                            </h3>
+                        </div>
+                        <div class="px-4 pb-4">
+                            <label for="{{ form.icon.id_for_label }}">{{ form.icon.label }}</label>
+                            <div class="flex my-2">
+                                {# djlint:off H006 #}
+                                <img id="poi-category-icon-preview"
+                                     class="w-10 rounded-l"
+                                     src="{{ form.instance.icon }}"
+                                     alt="{{ form.instance.icon }}"/>
+                                {# djlint:on #}
+                                <div class="relative flex-grow">{% render_field form.icon|add_error_class:"border-red-500" %}</div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="xl:w-1/2 mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
+                        <div class="w-full p-4 rounded bg-water-500">
+                            <h3 class="heading font-bold text-black">
+                                <i icon-name="tag" class="mr-2"></i> {% translate "Category color" %}
+                            </h3>
+                        </div>
+                        <div class="px-4 pb-4">
+                            <label for="{{ form.color.id_for_label }}">{{ form.color.label }}</label>
+                            <div class="flex my-2">
+                                <span id="poi-category-color-preview"
+                                      class="w-16 rounded-l"
```

Comment:
```suggestion
                <div class="mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
                    <div class="w-full p-4 rounded bg-water-500">
                        <h3 class="heading font-bold text-black">
                            <i icon-name="map-pin" class="mr-2"></i> {% translate "Map marker" %}
                        </h3>
                    </div>
                    <div class="px-4 pb-4 grid xl:grid-cols-2 gap-4">
                        <div>
                            <label for="{{ form.icon.id_for_label }}">{{ form.icon.label }}</label>
                            <div class="flex">
                                {# djlint:off H006 #}
                                <img id="poi-category-icon-preview"
                                     class="w-10 mr-2"
                                     data-base-path="{% get_static_prefix %}/svg/poi-category-icons/"
                                     src="{% get_static_prefix %}/svg/poi-category-icons/{{ form.instance.icon }}.svg"
                                     alt=""/>
                                {# djlint:on #}
                                <div class="relative flex-grow">
                                    {% render_field form.icon|add_error_class:"border-red-500" id="poi-category-icon-field" %}
                                </div>
                            </div>
                        </div>
                        <div>
                            <label for="{{ form.color.id_for_label }}">{{ form.color.label }}</label>
                            <div class="flex">
                                <span id="poi-category-color-preview"
                                      class="w-12 rounded-l"
```
Actually, thinking about this, I'd say we put this into the same box since both input values are regarding the map pin.